backend-processing.ts

```diff
@@ -45,6 +45,7 @@ export default class BackendProcessing {
     let con = await (this.create_connection_to_database())
     var results = papa.parse("#" + data, { header: true, comments: "#" });
     await (results.data.forEach(element => {
+      if (element["Name"]!=null)
```

Comment:
You should put spaces around operators. 
```js
if (term != null) {
```
https://standardjs.com/rules-en.html
>**Infix operators** must be spaced.

Reply:
Note that this could be undefined i.e, Name wasn't an included header

Reply:
Sounds like a tool like prettier could help here...